WordPress/src/main/res/layout/site_creation_intents_titlebar.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <ImageButton
+        android:id="@+id/backButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/negative_margin_medium"
+        android:backgroundTint="@color/transparent"
+        android:contentDescription="@string/navigate_back_desc"
+        android:src="@drawable/ic_arrow_left_white_24dp"
+        android:tint="?attr/colorOnSurface" />
+
+    <!--TODO: Rename ModalLayoutPickerHeader style to something more generic. -->
```

Comment:
It's open question for reviewer(s) if I should actually do this 😄 :
- Rename ModalLayoutPickerHeader style to something more generic.

It seems to be reusable without any additional work, which is why I thought of renaming it.

Reply:
Seems reasonable, though curious if you'd also intend to rename the other similar-named styles?

Reply:
> though curious if you'd also intend to rename the other similar-named styles?
Thank you for the feedback Matthew 🙇
I was thinking about it, but I want to stay under 300 lines of code with the PR and I think that will restrict the number of names I can change 😊

Reply:
Decided not to rename it and removed the code comment instead :)